js/guides.js

```diff
@@ -96,3 +96,14 @@ $(document).ready(function() {
     osLabelElement.text(osLabel);
   }
 });
+
+
+let mybutton = document.getElementById("myBtn");
+
+function topFunction() {
+  document.body.scrollTop = 0;
+  document.documentElement.scrollTop = 0;
```

Comment:
You can also use the `scrollTo` function: http://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo

Extra benefit: smooth scroll, so it's not a very sudden jump when you click the arrow.

```suggestion
  window.scrollTo({
    top: 0,
    left: 0,
    behavior: "smooth",
  });
```

Reply:
@tombruijn Thank You soo much for optimizing my code I got to learn a lot from you. i have done the changes